bot/main.py

```diff
@@ -23,37 +53,45 @@
 
 
 def get_users_from_discordid(user_id):
+    """
+    Finds users from the database, given their ID and returns
+    a list containing those users.
+    """
     users = list(db.users.find({"discordId": str(user_id)}))
     return users
 
 
 def is_verified(user_id):
-    return True if get_users_from_discordid(user_id) else False
+    """Checks if any user with the given ID exists in the DB or not."""
+    return len(get_users_from_discordid(user_id)) == 0
```

Comment:
```suggestion
    return len(get_users_from_discordid(user_id)) != 0
```
This function is support to return `False` if the list is empty, and `True` otherwise. So this update breaks code

Reply:
Sure, I'll fix that